--- C:/Users/eedel/AppData/Local/Temp/cta2.php-rev140.svn001.tmp.php	jue. mar.  5 16:44:57 2015
+++ C:/wamp/Fimex/fimex/frontend/views/maquinado/cta2.php	mar. mar. 10 15:32:32 2015
@@ -109,6 +109,20 @@ $this->registerJS("
 										showFooter: true,
 										rowStyler:formateo,
 										
+										collapsible:true,
+
+										rownumbers:true,
+										
+										view:groupview,
+										groupField:'producto',
+										groupFormatter:function(value,rows){
+											if (rows.length ==1 ){
+												return value;
+											}
+										   return value + ' - ' + rows.length + ' Operacion(es)';
+											},
+																	
+										
 										onClickCell:hold,
 										toolbar:tb,
 										queryParams: {
@@ -157,7 +171,10 @@ $this->registerJS("
 													}
 										
 										">Maquina</th>
+										<th data-options="field:'op',width:50,sortable:true">OP</th>
+										<th data-options="field:'minop',width:50,sortable:true">MinOP</th>
 										<th data-options="field:'Minutos',width:50,sortable:true">Min</th>
+										<th data-options="field:'setup',width:50,sortable:true">Setup(min)</th>
 									
 										<th data-options="field:'producto',width:200,sortable:true">Parte</th>
 										
@@ -367,12 +384,8 @@ data-options="
 				 if($('#<?php echo $id ?>').datagrid('getRows')[editIndex]['s3'] >  0 && s3 == '' ) s3 = "0";
 				 if($('#<?php echo $id ?>').datagrid('getRows')[editIndex]['s4'] >  0 && s4 == '' ) s4 = "0";
 				 
-				
+			 
 				 
-				 
-				 
-				 
-				 
 				 prioridad = $(ed_prio.target).numberbox('getValue');
 				 $('#<?php echo $id ?>').datagrid('getRows')[editIndex]['Hold'] = hold;
 				 $('#<?php echo $id ?>').datagrid('getRows')[editIndex]['maquina1'] = maq1;
